### DISCUSSION TITLE
Renaming options in User Accounts dialog in the future versions

### DISCUSSION THREAD

**Discussion by Ramon-Salazar** [BearWare/TeamTalk5#2089] [Ideas]
It'll be much better if these options in the user account dialog to be renamed for better understandings.

The "Active accounts" should be renamed to "Accounts", since it has both active and inactive accounts.

The word "Disabled" should be removed, only inactive since it's too clear.

Also: "add/update" should be renamed to "Validate/update", while "delete" should be "invalidate", since on the servers we are validating invalid accounts, not creating since every username is invalid unless if an administrator validate them on a server/servers.

What do you think about this idea @bear101

**Comment by bear101** [comment]
I have not heard that people have problems understanding the labels in User Accounts dialog. I think it will cause more confusion to change the label and invalidate all the translations.

**Reply by Ramon-Salazar** [reply]
Probably because it might not matter for the most of people so they don't tend to complain about it but in general they have problems of understanding these labels especially the native English speakers.

Regarding loging in to an account for example which isn't exist it shows "invalid user account" instead of account not found, so instead of "add" it should be "validate, since that's the actual right word for validation of an invalid accounts while "Delete" is "Invalidate".


The translations can be  updated normally, if the translators are too active since what I've noticed most of them they are.

**Comment by jimmytruth** [comment]
Yea I agree with Ramon, these options should be renamed for good since it'll be clearer, and he's  is absolutely right regarding translations @bear101
